pandas/tools/pivot.py

```diff
@@ -393,6 +393,12 @@ def crosstab(index, columns, values=None, rownames=None, colnames=None,
         Add row/column margins (subtotals)
     dropna : boolean, default True
         Do not include columns whose entries are all NaN
+    normalize : boolean, or {'all','index','columns'}, default False
+        Normalize by dividing all values by the sum of values.
+        If passed `all` or `True`, will normalize over all values.
```

Comment:
put a blank line before the ``If passed`` and use ``-`` (e.g. a bulleted list) for each